apps/meteor/app/assets/server/assets.ts

```diff
@@ -466,29 +488,29 @@ Meteor.methods({
 
 WebApp.connectHandlers.use(
 	'/assets/',
-	Meteor.bindEnvironment(function (req, res, next) {
+	Meteor.bindEnvironment((req: Request, res: Response, next: NextFunction) => {
 		const params = {
 			asset: decodeURIComponent(req.url.replace(/^\//, '').replace(/\?.*$/, '')).replace(/\.[^.]*$/, ''),
 		};
 
-		const file = assets[params.asset] && assets[params.asset].cache;
+		const file = getAssetByKey(params.asset) && getAssetByKey(params.asset).cache;
```

Comment:
why not store `getAssetByKey(params.asset)` in a variable and use it in the rest of the function? you checking if it has values in a lot of places